colossalai/inference/struct.py

```diff
@@ -46,6 +47,17 @@ def is_waiting(status: "RequestStatus") -> bool:
         return status == RequestStatus.WAITING
 
 
+@dataclass
+class Sequence_:
```

Comment:
Same as above. The naming is causing confusion; please use a meaningful and clear name. You could even create a baes sequence class with commonly used attrs such as request id and prompt, and modify, for example, the `DiffusionSequence` and `LLMSequence`,  to inherit from the base class.

Reply:
thanks!